src/main/java/com/cms/claimmanagement/insuranceCompany/policy/Service/PolicyService.java

```diff
@@ -20,34 +21,39 @@ public class PolicyService {
     @Autowired
     public PolicyRepository policyRepository;
 
-    public List<PolicyResponseData> listPolicies(){
-        return policyRepository.findAll().stream().map(PolicyResponseData::new).toList();
+    public List<PolicyResponseDTO> listPolicies(){
+        List<PolicyResponseDTO> listedPolicies = policyRepository
+                .findAll()
+                .stream()
+                .map(PolicyEntity::getPolicyNo)
```

Comment:
Why is this .map(PolicyEntity::getPolicyNo) used?

Reply:
the same mistake, supposed to set the policyNo to policyResponseDTO